subsys/bluetooth/host/smp.c

```diff
@@ -1927,6 +1935,17 @@ static void smp_timeout(struct k_work *work)
 	atomic_set_bit(smp->flags, SMP_FLAG_TIMEOUT);
 }
 
+#if defined(CONFIG_BT_SETTINGS)
+static void bt_smp_save_ccc_and_cf_on_pairing_complete(struct k_work *work)
+{
+	struct bt_smp *smp = CONTAINER_OF(work, struct bt_smp, work_settings_ccc_and_cf_save);
+
+	bt_gatt_identity_resolved(smp->chan.chan.conn, &(smp->chan.chan.conn->le.init_addr),
+				  &smp->chan.chan.conn->le.dst);
+}
+
+#endif /* CONFIG_BT_SETTINGS */
```

Comment:
Let's not do this. There is an `identity_resolved` callback that's part of the `bt_conn_cb` struct. I think GATT should register one of these structs and use that mechanism to get notified of identity resolving.

Reply:
Yes. That sounds better to me too, but I would prefer if this could be done in another PR.